### PR TITLE
DOCS-7946: Add instructions for custom frameworks

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do? What is the motivation?

DOCS-7946.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/cloud_security_management/misconfigurations/frameworks_and_benchmarks/custom_frameworks.md [hunk 1]
```diff
@@ -15,16 +14,33 @@ further_reading:
...
+Next, add requirements to the framework:
 
-- `framework`, `requirement`, and `control` are reserved keys and cannot be used in custom tags. See [Custom Rules][3] for more information.
-- You can also add custom tags to rules using the [Update an existing rule API endpoint][4].
+<div class="alert alert-warning">You must add at least one requirement before you can save the custom framework.</div>
+
+1. Click **Add Requirement**.
+1. Enter the following details:
+    - **Requirement**: A requirement acts as a control family, enabling you to add controls and associate rules with each control.
+    - **Control**: A control represents the criteria that the requirement must meet and includes the rules associated with these criteria.
```
[the remark this answers is not in the document]
Comment (drichards-87) [reply content/en/security/cloud_security_management/misconfigurations/frameworks_and_benchmarks/custom_frameworks.md:34]: Thanks, I'll add that to the PR.

File: content/en/security/cloud_security_management/misconfigurations/frameworks_and_benchmarks/custom_frameworks.md [hunk 2]
```diff
@@ -15,16 +14,33 @@ further_reading:
...
+With custom frameworks, you can define and measure compliance against your own cloud security baseline. Custom frameworks are listed on the Cloud Security Management (CSM) [Compliance][6] page, have their own real-time report and [security posture score][7], and are queryable within explorers and dashboards.
 
-1. On the [**Misconfiguration Rules**][5] page, select the rule you wish to add the custom tag to.
-2. Under **Say what's happening**, navigate to the **Tag resulting misconfigurations with** section and add the `key:value` for the custom tag.
-3. Click **Update Rule**.
+1. On the [CSM Compliance page][6], click **Create Framework**.
+1. Enter the following details:
+    - **Framework name**: The name of your framework. Can include characters, numbers, and spaces. Must be at least five characters long.
+    - **Handle**: The tag name for the custom framework. Can include lowercase letters, numbers, dashes, underscores, and periods. This value will be used for querying the framework in the explorer or dashboards.
```
Comment (alai97) [inline content/en/security/cloud_security_management/misconfigurations/frameworks_and_benchmarks/custom_frameworks.md:22]:
```suggestion
    - **Handle**: The tag name for the custom framework. Can include lowercase letters, numbers, dashes, underscores, and periods. This value is used to query the framework in the explorer or in dashboards.
```

File: content/en/security/cloud_security_management/misconfigurations/frameworks_and_benchmarks/custom_frameworks.md [hunk 3]
```diff
@@ -15,16 +14,33 @@ further_reading:
...
+    - **Version**: The version of the framework. Can include lowercase letters, numbers, dashes, underscores, and periods.
+    - **Image URL**: A publicly accessible URL for an image that will be used to identify the framework.
```
Comment (alai97) [inline content/en/security/cloud_security_management/misconfigurations/frameworks_and_benchmarks/custom_frameworks.md:24]:
```suggestion
    - **Image URL**: A publicly accessible URL for an image that is used to identify the framework.
```

File: content/en/security/cloud_security_management/misconfigurations/frameworks_and_benchmarks/custom_frameworks.md [hunk 4]
```diff
@@ -15,16 +14,33 @@ further_reading:
...
+1. Click **Next Step: Create Your Framework**.
 
-**Notes**:
+Next, add requirements to the framework:
 
-- `framework`, `requirement`, and `control` are reserved keys and cannot be used in custom tags. See [Custom Rules][3] for more information.
-- You can also add custom tags to rules using the [Update an existing rule API endpoint][4].
+<div class="alert alert-warning">You must add at least one requirement, control, and rule before you can save the custom framework.</div>
```
Comment (alai97) [inline content/en/security/cloud_security_management/misconfigurations/frameworks_and_benchmarks/custom_frameworks.md:29]:
```suggestion
<div class="alert alert-warning">You must add at least one requirement, control, and rule before saving the custom framework.</div>
```

**Review by alai97 — APPROVED** [review]
LGTM, small suggestions.

**Pull request merged**